{"name": "Verify scan details API returns 400 for invalid scan ID format -- @1.2 ", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyD08yxn7V_YAfkAY8mKDTYjfKV_qoPrdRQ\n", "trace": "  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\FireBase_get_a_specific_sca_by_id.py\", line 22, in step_impl\n    id_token = get_firebase_id_token()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\utilities\\authentication.py\", line 16, in get_firebase_id_token\n    response.raise_for_status()  # Raise an error for non-2xx responses\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "steps": [{"name": "Given FireBase_User sends GET request to fetch scan details for scan ID \"' OR 1=1 --\"", "status": "broken", "statusDetails": {"message": "requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyD08yxn7V_YAfkAY8mKDTYjfKV_qoPrdRQ\n", "trace": "  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\FireBase_get_a_specific_sca_by_id.py\", line 22, in step_impl\n    id_token = get_firebase_id_token()\n               ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\utilities\\authentication.py\", line 16, in get_firebase_id_token\n    response.raise_for_status()  # Raise an error for non-2xx responses\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\requests\\models.py\", line 1026, in raise_for_status\n    raise HTTPError(http_error_msg, response=self)\n"}, "start": 1757682191901, "stop": 1757682192433}, {"name": "When FireBase_the user receives the scan details response", "status": "skipped", "start": 1757682192436, "stop": 1757682192436}, {"name": "Then FireBase_the status code returned should be 400 for bad request error", "status": "skipped", "start": 1757682192436, "stop": 1757682192436}, {"name": "And FireBase_the response should contain an error message \"Bad Request\"", "status": "skipped", "start": 1757682192436, "stop": 1757682192436}, {"name": "And FireBase_the message should contain \"Invalid input:\"", "status": "skipped", "start": 1757682192436, "stop": 1757682192436}], "parameters": [{"name": "scan_id", "value": "' OR 1=1 --"}], "start": 1757682191900, "stop": 1757682192436, "uuid": "5b94523e-d89d-4ba9-8253-7118e6d6c80b", "historyId": "3f63b7e85b05a662bf1b72d66fd3e3c1", "fullName": "Get Scan Details using Genr3d API: Verify scan details API returns 400 for invalid scan ID format", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Negative"}, {"name": "tag", "value": "Security"}, {"name": "tag", "value": "Test"}, {"name": "feature", "value": "Get Scan Details using Genr3d API"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}], "titlePath": ["features", "Get Scan Details using Genr3d API"]}